contracts/http/response.go

```diff
@@ -49,6 +49,11 @@ type Response interface {
 	Render() error
 }
 
+type AbortResponse interface {
+	Response
+	Abort() error
```

Comment:
Any concerns about not adding `Abort` to `Response` directly?

Reply:
I decided not to implement the abort method for streaming, downloads, and other formats as it was unnecessary. Currently, it is implemented only for `StringResponse` and `JsonResponse`, which makes more sense to me.

Reply:
I think `Data` and `Redirect` are needed, too.